configs/platforms/sles-15-x86_64.rb

```diff
@@ -0,0 +1,9 @@
+platform "sles-15-x86_64" do |plat|
+  plat.servicedir "/usr/lib/systemd/system"
+  plat.defaultdir "/etc/sysconfig"
+  plat.servicetype "systemd"
+
+  plat.provision_with "zypper -n --no-gpg-checks install -y aaa_base autoconf automake rsync gcc make rpm-build libboost_atomic1_66_0-devel libboost_chrono1_66_0-devel libboost_container1_66_0-devel libboost_date_time1_66_0-devel libboost_filesystem1_66_0-devel libboost_graph1_66_0-devel libboost_iostreams1_66_0-devel libboost_locale1_66_0-devel libboost_log1_66_0-devel libboost_math1_66_0-devel libboost_program_options1_66_0-devel libboost_random1_66_0-devel libboost_regex1_66_0-devel libboost_serialization1_66_0-devel libboost_signals1_66_0-devel libboost_system1_66_0-devel libboost_test1_66_0-devel libboost_thread1_66_0-devel libboost_timer1_66_0-devel libboost_wave1_66_0-devel gettext-tools cmake yaml-cpp-devel"
```

Comment:
Nitpick, but could these packages be formatted like https://github.com/puppetlabs/puppet-runtime/blob/master/configs/platforms/sles-12-x86_64.rb#L7 ?

Reply:
The way it's formatted currently is consistent with the other platform configs in this repo, so it might be better to change them all at once in a separate PR. I will say though that it's been handy to have the package list all on one line as I've needed to copy/paste it into a terminal frequently during development. I suppose that won't matter though once the platform is finalized.

Reply:
Ok that makes sense. At least everything's in one place.